.buildkite/steps/release-github.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 set -e
 
-GITHUB_RELEASE_IMAGE="buildkite/github-release@sha256:e5ae9753a8246ace67f3669baa63103429367c999bfda0a227c91a4ebf34c23f"
```

Comment:
Is this now included in the container?

Reply:
yup - deploytools has github-release v1.0